sdk/create-dapp/templates/react-client-dapp/src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import { ConnectButton } from "@mysten/dapp-kit";
+import { Box, Container, Flex, Heading } from "@radix-ui/themes";
```

Comment:
Oh I see why you didn't use Tailwind

Reply:
I'm fine with this because it makes it easier to spin out examples

Reply:
Yeah, I wanted the code to be as simple/readable as possible.  It's easy to rip out